lib/ex_webrtc/rtp_transceiver.ex

```diff
@@ -89,6 +91,15 @@ defmodule ExWebRTC.RTPTransceiver do
   @spec from_mline(ExSDP.Media.t(), non_neg_integer(), Configuration.t()) :: t()
   def from_mline(mline, mline_idx, config) do
     codecs = get_codecs(mline, config)
+
+    if codecs == [] do
+      rtpmap = ExSDP.get_attribute(mline, :rtpmap)
+
+      Logger.warning(
+        "No valid codecs for \"#{rtpmap}\" found in registered %ExWebRTC.PeerConnection.Configuration{}"
```

Comment:
I think we should use inspect here, shouldn't we? rtpmap should be a struct.

```suggestion
        "No valid codecs for #{inspect(rtpmap)} found in registered %ExWebRTC.PeerConnection.Configuration{}"
```

Reply:
This actually prints nicely by default thanks to: https://github.com/membraneframework/ex_sdp/blob/master/lib/ex_sdp/attribute/rtp_mapping.ex#L62

Let me know if you want to keep that, or swap it out for inspect!